Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -123,14 +149,42 @@ fun Home(
     BackHandler(enabled = navigator.canNavigateBack()) {
         navigator.navigateBack()
     }
+
+    val homeState = HomeState(
+        windowSizeClass = windowSizeClass,
```

Comment:
NIT: could be simplified by having just `isCompact: Boolean`, which would also potentially recompose less when changing the window size